hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/precompile/impl/AbstractAssociatePrecompile.java

```diff
@@ -75,12 +74,7 @@ protected AbstractAssociatePrecompile(
 
     @Override
     public void run(final MessageFrame frame) {
-        // --- Check required signatures ---
-        final var accountId =
-                Id.fromGrpcAccount(Objects.requireNonNull(associateOp).accountId());
-        final var hasRequiredSigs = KeyActivationUtils.validateKey(
-                frame, accountId.asEvmAddress(), sigsVerifier::hasActiveKey, ledgers, aliases, TokenAssociateToAccount);
-        validateTrue(hasRequiredSigs, INVALID_FULL_PREFIX_SIGNATURE_FOR_PRECOMPILE, ASSOCIATE_FAILURE_MESSAGE);
+        Objects.requireNonNull(associateOp);
```

Comment:
Q: will this break contracts on the old model?

Reply:
Answering my own question.
Old model isn't affected since non facade flows will go through `AssociatePrecompile` and `DissociatePrecompile` where you moved this logic to? 

This would have been a good note in the PR description

Reply:
Good point.  I'll add that to the description